RevoUIComponents/src/Views/AsyncButton.swift

```diff
@@ -29,10 +30,12 @@ public class AsyncButton : UIButton {
     public func animateProgress(){
         isEnabled = false
         removeCompletionImage()
+        shouldOverrideTitle = false
```

Comment:
quin sentit te afegir el shouldOverride? si es posa a falsetrue, simplement aquí i s'esta cridant sobre?

Reply:
Perqué, de moment, només interessa que no es sobrescrigui el titol, !nomes en aquest tram d'aquesta funció!. I com s'utilitza en el setTitle(), més a baix, per això el crear la variable a fora de la funció. Cóm ho faries tu?

Reply:
si nomes es en aquest tram, simplement no cridar la funcio 
ara bé, perque no vols que es posi buit? la idea es que quan s'apreta s'amaga el títol i surt la rodoneta de pensar

Reply:
Doncs perqué tal i com està, el text se sobrescriu i quan hi ha un error i fa l 'animació, després el boto queda sense el text. De fet però, si trec aquestes 4 linies:
        setTitle("",  for: .normal)
        setImage(nil, for: .normal)
        setTitle("",  for: .disabled)
        setImage(nil, for: .disabled)
dóna el mateix resultat que afegir-hi el flag.